core/chains/evm/txmgr/txmgr.go

```diff
@@ -156,7 +154,7 @@ func NewTxm(db *sqlx.DB, ethClient evmclient.Client, cfg Config, keyStore KeySto
 		reset:            make(chan reset),
 	}
 	if cfg.EthTxResendAfterThreshold() > 0 {
-		b.ethResender = NewEthResender(lggr, orm, ethClient, keyStore, defaultResenderPollInterval, cfg)
+		b.ethResender = NewEthResender(lggr, NewORM(b.db, b.logger, b.config), ethClient, keyStore, defaultResenderPollInterval, cfg)
```

Comment:
Inside the TxMgr, for all sub-components, we should be consistent with the following:
- Who creates the ORM. TxMgr, or each sub-component creates its own?
- Who owns closing the ORM at the end.

I see the confirmer, resender are being passed an ORM from the TxMgr, and they are expected to close it.
However the Broadcaster creates its own ORM, and closes it.

Can you pick one strategy, and consistently apply it in all components? Makes it easy and uniform across.

Reply:
Yeah, the broadcaster will also be passed in the ORM and will close it. I haven't updated the broadcaster because I haven't gotten to those changes yet (next PR).